resources/function-controller/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,41 @@
+# in case of changes please test this on GKE, Azure, Gardener and Minikube
+kind: ConfigMap
+metadata:
+  name: {{ template "registry-fullname" }}-cert-patch
+  namespace: istio-system
+apiVersion: v1
+data:
+  patch-docker.sh: |
+    #!/usr/bin/env bash
```

Comment:
Maybe inject script by Helm template?

Reply:
this configmap is almost just this script, there's no point in extracting this to some kind of helper, it would be convoluted